File: go/bot/getQuote.go
package bot

import (
	"encoding/json"
	"io"
	"net/http"
)

type Quote struct {
	Q string `json:"q"`
	A string `json:"a"`
}

func GetQuote() string {
	var quote []Quote
	res, _ := http.Get("https://zenquotes.io/api/random")
	data, _ := io.ReadAll(res.Body)
	json.Unmarshal(data, &quote)

	return "\"" + quote[0].Q + "\"\n-" + quote[0].A
}

File: python/get_quote.py
import requests

def get_quote():
    raw = requests.get("https://zenquotes.io/api/random")
    # Our API returns a list of length 1, so we have to get the data from an array index
    res = raw.json()[0]
    return f'\"{res["q"]}\"\n-{res["a"]}'